src/main/java/uk/gov/digital/ho/hocs/workflow/client/infoclient/dto/StageTypeDto.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.digital.ho.hocs.workflow.client.infoclient.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@NoArgsConstructor
+@Getter
+public class StageTypeDto implements Serializable {
+
+    //TODO: remove - used in UI

```

Comment:
What would be the effort required to remove these now? Given they've been highlighted here?